homeassistant/components/airly/sensor.py

```diff
@@ -120,8 +121,10 @@ class AirlySensorEntityDescription(SensorEntityDescription):
         key=ATTR_API_CO,
         device_class=SensorDeviceClass.CO,
         name=ATTR_API_CO,
-        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
```

Comment:
IMO we should use µg/m³ as the Airly website does and remove device class. CO concentrations in ppm are less readable.

Reply:
> CO concentration in ppm is less readable.

That is opinionated.
We should always choose a standard across multiple integrations over personal opinion in a single instance.

Reply:
> CO concentration in ppm is less readable.

In the context of outside air quality I agree, if we'd have to use part-per-x unit format, ppb would be better suited for the scale of values returned.

> We should always choose a standard across multiple integrations over personal opinion in a single instance

Also agree, I'm partial to your suggestion of per-entity user-adjustable units in home-assistant/architecture#682 and will add to that discussion.